### DISCUSSION TITLE
Change Highlight colors to match vscode-vim

### DISCUSSION THREAD

**Discussion by stevenmqnguyen** [folke/flash.nvim#178] [Q&A] [answered]
The colors for flash.nvim are currently [hardcoded](https://github.com/folke/flash.nvim/blob/967117690bd677cb7b6a87f0bc0077d2c0be3a27/lua/flash/highlight.lua#L10) for vscode environments. How can I change these to match my gruvbox theme in vscode using vscode-neovim? I'm currently using flash.nvim as part of LazyVim.

Is there a path forward to getting this to work out of the box, or what work needs to be done to achieve this? related #58

**Comment by IndianBoy42** [comment, marked as answer]
You can override those hl groups with `nvim_set_hl` or `:hi` 

Just do it after flash setup
